protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -298,3 +298,17 @@ enum CoreOperatorType {
   NESTED_LOOP_JOIN = 35;
   AVRO_SUB_SCAN = 36;
 }
+
+message Func {
+  optional string name = 1;
```

Comment:
This is the name of the UDF class? Presumably the full name with package prefix?